dcpandas/df_builder.py

```diff
@@ -0,0 +1,292 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data Commons Pandas API DataFrame Builder Module.
+
+Provides functions for building pandas DataFrames using the Data Commons Graph.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import pandas as pd
+import six
+
+import datacommons.stat_vars as dc
+
+
+def build_time_series(place, stat_var):
+    """Constructs a pandas Series with `dates` as the index and corresponding `stat_var` statistics as values.
+    
+    Args:
+      place (`str`): The dcid of Place to query for.
+      stat_var (`str`): The dcid of the StatisticalVariable.
+    Returns:
+      A pandas Series with Place IDs as the index, and Observed statistics as values.
+    """
+    return pd.Series(dc.get_stat_series(place, stat_var))
+
+
+def _group_stat_all_by_obs_options(places, stat_vars, mode):
```

Comment:
is it better to have a default "mode"?

Reply:
SG, was keeping it string in case we have other modes, but we can cross that bridge later. Changed.